package.json

```diff
@@ -62,6 +62,10 @@
     {
       "path": "logger.js",
       "limit": "135 B"
+    },
+    {
+      "path": "devtools.js",
+      "limit": "500 B"
```

Comment:
You can remove it :). It is not important for a development build (you can remove `logger.js` as well, I just forgot to remove it :D).